### ISSUE TITLE
Firefox visual bug on cutter

### ISSUE THREAD

**Issue by SocialHulkHogan** [WheatonCS/Lexos#349]
on the cutter page, removing whitespace makes the fieldset class="individualpreviewwrapper" element WAY too wide, no longer fitting in the preview panel div, causing some buttons to appear offscreen. I have no idea why this happens, and it only happens in Firefox.

**Comment by chantisnake** [comment]
https://github.com/WheatonCS/Lexos/issues/347#issuecomment-226231797

I think this is a duplicate

**Comment by SocialHulkHogan** [comment]
I think your mom is a duplicate. Close the other one this is more detailed

**Comment by scottkleinman** [comment]
I cannot reproduce this bug. It works for me in Firefox and Chrome. I used the beginning of _Pride and Prejudice_. After scrubbing all white space, I ended up with a long string of characters that was wider than the preview div. This produced a scroll bar at the bottom. It's not pretty, but it works.

If the idea is to avoid this, I have introduced a possible fix. However, it may cause problems for when we don't strip white space. If that's the case, we'll have to introduce the class rules dynamically.

Let me know if that works, or if I've misinterpreted the scenario producing the bug.
